pycode/epidemiology/epidata/getRKIDatawithEstimations.py

```diff
@@ -210,8 +210,10 @@ def compare_estimated_and_rki_deathsnumbers(df_jh, df_rki, data_path, read_data,
     if not read_data:
         download_weekly_deaths_numbers_rki(data_path)
 
-    df_real_deaths_per_week=gd.loadExcel("RKI_deaths_weekly", apiUrl=data_path,
-                                         extension=".xlsx", sheet_name='COVID_Todesfälle')
+    df_real_deaths_per_week = gd.loadExcel(
```

Comment:
This file is something that hasn't been maintained for a long time. This should be checked in particular in a next step (new issue).